### PR TITLE
Add portuguese language (pt-PT)

### PR DESCRIPTION
**Pull request by horus68**
Added Portuguese (Portugal) - pt-PT language code

## Summary by CodeRabbit

- **New Features**
	- Added Portuguese (pt_PT) locale to the translations system.
	- Updated the Language Switcher to include Portuguese as a language option.
	- Introduced a comprehensive JSON file for Portuguese translations covering various user interface elements.

- **Bug Fixes**
	- No bug fixes were implemented in this release. 

- **Documentation**
	- Enhanced localization documentation for Portuguese-speaking users.

### CODE REVIEW

**Comment by horus68** [comment]
@NayamAmarshe I've removed the previous PR, so no more merge conflicts

File: renderer/locales/pt_PT.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+      "WARNING": "Tudo o que seja acima de 5X pode causar problemas de desempenho em alguns dispositivos!",
+      "ADDITIONAL_WARNING": "Isto pode causar problemas de desempenho em alguns dispositivos!"
+    },
+    "LOG_AREA": {
+      "ON_COPY": "COPIADO ✅",
+      "BUTTON_TITLE": "COPIAR HISTORIAIS 📋",
+      "NO_LOGS": "Sem historiais para exibir"
+    },
+    "OVERWRITE_TOGGLE": {
+      "TITLE": "SUBREPOR À IMAGEM ANTERIOR",
```
Comment (NayamAmarshe) [inline renderer/locales/pt_PT.json:59]: SOBREPOR?

File: renderer/locales/pt_PT.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      "SELECT_IMAGE": "Selecionar imagem",
+      "SELECT_FOLDER_DESCRIPTION": "Certifique-se de que a pasta não contém nada, exceto imagens PNG, JPG, JPEG ou WEBP.",
+      "SELECT_IMAGES_DESCRIPTION": "selecione ou arraste e solte uma imagem PNG, JPG, JPEG ou WEBP.",
+      "PASTE_IMAGE_DESCRIPTION": "Prima Ctrl+V ou ⌘+V para colar a imagem a partir da área de transferência"
+    },
+    "PROGRESS": {
+      "PROCESSING_TITLE": "A processar a imagem...",
+      "SCALING_CONVERTING_TITLE": "A escalar e converter a imagem...",
+      "WAIT_TITLE": "Aguarde...",
+      "SUCCESS_TITLE": "Aprimoramento com sucesso!",
```
Comment (NayamAmarshe) [inline renderer/locales/pt_PT.json:198]: "Aprimorado com sucesso!"?

File: renderer/locales/pt_PT.json (hunk 1 above)
Comment (horus68) [reply renderer/locales/pt_PT.json:59]: Agree, it is a typo!

File: renderer/locales/pt_PT.json (hunk 2 above)
Comment (horus68) [reply renderer/locales/pt_PT.json]: agree!

**Comment by NayamAmarshe** [comment]
Thanks, merging :)

**Pull request merged**